nucypher/blockchain/eth/scripts/estimate_gas_tester.py

```diff
@@ -5,289 +5,486 @@
-    tx = escrow.transact({'from': ursula2}).deposit(1, 1)
-    chain.wait.for_receipt(tx)
-    print("Third deposit tokens again = " + str(escrow.contract.estimateGas({'from': ursula3}).deposit(1, 1)))
-    tx = escrow.transact({'from': ursula3}).deposit(1, 1)
-    chain.wait.for_receipt(tx)
+    print("First deposit tokens again = " +
+          str(miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))
+              .estimateGas({'from': ursula1})))
+    tx = miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))\
+        .transact({'from': ursula1})
+    testerchain.wait_for_receipt(tx)
+    print("Second deposit tokens again = " +
+          str(miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))
+              .estimateGas({'from': ursula2})))
+    tx = miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))\
+        .transact({'from': ursula2})
+    testerchain.wait_for_receipt(tx)
+    print("Third deposit tokens again = " +
+          str(miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))
+              .estimateGas({'from': ursula3})))
+    tx = miner_agent.contract.functions.deposit(constants.MIN_ALLOWED_LOCKED * 2, int(constants.MIN_LOCKED_PERIODS))\
+        .transact({'from': ursula3})
+    testerchain.wait_for_receipt(tx)
 
     # Wait 1 period and mint tokens
-    chain.time_travel(1)
-    print("First mining again = " + str(escrow.contract.estimateGas({'from': ursula1}).mint()))
-    tx = escrow.transact({'from': ursula1}).mint()
-    chain.wait.for_receipt(tx)
-    print("Second mining again = " + str(escrow.contract.estimateGas({'from': ursula2}).mint()))
-    tx = escrow.transact({'from': ursula2}).mint()
-    chain.wait.for_receipt(tx)
-    print("Third/last mining again = " + str(escrow.contract.estimateGas({'from': ursula3}).mint()))
-    tx = escrow.transact({'from': ursula3}).mint()
-    chain.wait.for_receipt(tx)
+    testerchain.time_travel(periods=1)
+    print("First mining again = " + str(miner_agent.contract.functions.mint().estimateGas({'from': ursula1})))
+    tx = miner_agent.contract.functions.mint().transact({'from': ursula1})
+    testerchain.wait_for_receipt(tx)
+    print("Second mining again = " + str(miner_agent.contract.functions.mint().estimateGas({'from': ursula2})))
+    tx = miner_agent.contract.functions.mint().transact({'from': ursula2})
+    testerchain.wait_for_receipt(tx)
+    print("Third/last mining again = " + str(miner_agent.contract.functions.mint().estimateGas({'from': ursula3})))
+    tx = miner_agent.contract.functions.mint().transact({'from': ursula3})
+    testerchain.wait_for_receipt(tx)
 
     # Create policy
-    policy_id_1 = os.urandom(20)
-    policy_id_2 = os.urandom(20)
+    policy_id_1 = os.urandom(16)
```

Comment:
Why 16 bytes specifically? Let's maybe get a constant here so if it changes, we have only one place to change it.